tests/test_mpi.py

```diff
@@ -147,29 +169,30 @@ def test_halo_exchange_bilateral_asymmetric(self):
             0 0 0 0 0 0 0     0 0 0 0 0 0 0
         """
         grid = Grid(shape=(12, 12))
-        f = Function(name='f', grid=grid, space_order=(1, 2, 1))
+        x, y = grid.dimensions
 
-        distributor = grid.distributor
-        f.data[:] = distributor.myrank + 1
+        f = Function(name='f', grid=grid, space_order=(1, 2, 1))
```

Comment:
remind me what tuple space_order is?

Reply:
(space_order, halo_left_extent, halo_right_extent)